### DISCUSSION TITLE
Apex Mobile Support

### DISCUSSION THREAD

**Discussion by biast12** [gildor2/UEViewer#269] [Ideas]
ok so Apex Mobile just came out a few days ago, and we're a group of people who wanna get the exclusive legends and skins model / files, and when we tried to figure out how we could get it did we find out that the compression algorithm isn't supported by UEViewer so we wanna ask if someone with more experience than us want to look into it? would really help a lot of people since they're adding exclusive things to that game = we can't just get it from the main game

**Comment by gildor2** [comment]
Removed content sharing links, I don't want to get any legal problems with this page.
Probably you're talking about this game?
https://www.gildor.org/smf/index.php/topic,7731.0.html

**Reply by biast12** [reply]
Fair just wanted to give you guys all the info i could to maybe make it easier, but no, that was for an earlier version of Apex Mobile, it looks like they did something new for the full release, but the umodel_morphs_export_mobile and the umodel_apex_legends_v2 versions of umodel doesn't work anymore after the full release

**Reply by gildor2** [reply]
You should ask the the forum then, in that thread. That was a custom build, I didn't author it.

**Reply by biast12** [reply]
fair, just thought it could be something useful to add, but i guess not
